src/iop/negadoctor.c

```diff
@@ -0,0 +1,1387 @@
+/*
+    This file is part of darktable,
```

Comment:
Header should now be:

  This file is part of darktable,
  Copyright (C) 2020 darktable developers.